scripts/thanks.js

```diff
@@ -0,0 +1,33 @@
+if (!remoteUrl) {
+  var remoteUrl = "https://api.carpoolvote.com/live";
+}
+
+const handleBackendResponse = () => {
+  var data = tinyQuery.getAll();
```

Comment:
Seeing as this depends on tinyQuery (😀), we'll need to ensure that this script gets added to footer.html after tinyQuery is included, to ensure that there aren't any dependency issues.